src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -145,6 +145,16 @@ class CompanyStep extends React.Component {
     validate() {
         const errors = {};
 
+        if (this.state.manualAddress) {
+            if (!isValidAddress(this.state.addressStreet)) {
+                errors.addressStreet = true;
+            }
+
+            if (!isValidZipCode(this.state.addressZipCode)) {
+                errors.addressZipCode = true;
+            }
```

Comment:
Can probably condense this to the following, unless you want `errors.addressStreet` to be `undefined` instead of `false` after this runs.

```suggestion
            errors.addressStreet = !isValidAddress(this.state.addressStreet) || !isValidZipCode(this.state.addressZipCode);
```

Reply:
? I'm a bit confused. Why do you want to get rid of `errors.addressZipCode`?